app/models/miq_queue.rb

```diff
@@ -179,6 +179,12 @@ def self.put_deprecated(*args)
     put(*args)
   end
 
+  # Place holder for deprecated put_unless_exists call
+  # which might not be needed in the new architecture
+  def self.put_unless_exists_deprecated(*args)
```

Comment:
Why not deprecate `put_unless_exists`?  Are there other cases that aren't deprecated?